lib/my_home_page.dart

```diff
@@ -341,6 +341,7 @@ class _MyHomePageState extends ConsumerState<MyHomePage> {
       appBar: SolidAppBarConfig(
         title: _menuItems[_selectedIndex].title,
         versionConfig: SolidVersionConfig(
+          version: '0.0.12+7',
```

Comment:
Why is the hard coded version back in here? SolidVersionConfig should not need it anymore.